template/deployments/helm/charts/templates/deployment.yaml

```diff
@@ -56,6 +56,9 @@ spec:
               port: http
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+          envFrom:
+            - configMapRef:
+                name: {{ .Values.configMapName }}
```

Comment:
this needs to be the same name as the actual configmap name, you can call it something like `envvars-configmap`